README.md

```diff
@@ -79,17 +87,39 @@ See [the wiki](https://github.com/Parsely/wp-parsely/wiki/Setting-up-a-WP-plugin
 
 ## Screenshots
 
-1. The main settings screen of the wp-parsely plugin  
+1. The main settings screen of the WP-Parsely plugin  
 ![The main settings screen of the wp-parsely plugin](.wordpress-org/screenshot-1.png)
 
-2. The standard JavaScript include being inserted before `</body>`
-![2. The standard JavaScript include being inserted before body tag](.wordpress-org/screenshot-2.png)
+2. The settings for the Parse.ly Recommended Widget.  Engage your visitors with predictive and personalized recommendations from Parse.ly. 
+![The settings for the Parse.ly Recommended Widget](.wordpress-org/screenshot-2.png)
+   
+3. A view of the Parse.ly Dashboard Overview. Parse.ly offers analytics that empowers you to better understand how your content is peforming.
+![The Parsely Dashboard Overview](.wordpress-org/screenshot-3.png)
+
+## Sample Parse.ly metadata
+
+The standard Parse.ly JavaScript tracker inserted before the closing `body` tag:
+
+    <!-- START Parse.ly Include: Standard -->
+
+       <script data-cfasync="false" id="parsely-cfg" data-parsely-site="wpvip.com" src="//cdn.parsely.com/keys/wpvip.com/p.js"></script>
+
+    <!-- END Parse.ly Include: Standard -->
```

Comment:
This bit isn't the metadata, so it should be moved outside of the `Sample Parse.ly metadata` section (can have its own heading).

Also:

```suggestion
The standard Parse.ly JavaScript tracker inserted before the closing `body` tag:

    <!-- START Parse.ly Include: Standard -->

       <script data-cfasync="false" id="parsely-cfg" data-parsely-site="example.com" src="https://cdn.parsely.com/keys/example.com/p.js"></script>

    <!-- END Parse.ly Include: Standard -->
```